spec/cc/engine/analyzers/php/main_spec.rb

```diff
@@ -2,12 +2,17 @@
 require 'cc/engine/analyzers/php/main'
 require 'cc/engine/analyzers/reporter'
 require 'cc/engine/analyzers/engine_config'
+require 'cc/engine/analyzers/file_list'
 require 'flay'
 require 'tmpdir'
 
 module CC::Engine::Analyzers::Php
   describe Main do
-    before { @code = Dir.mktmpdir }
+    before do
+      @code = Dir.mktmpdir
+      Dir.chdir(@code)
```

Comment:
I'd be all for migrating to RSpec though, if that's something we want to do so we can have nice things like `around`.